internal/dcache/cluster_manager/cluster_manager_impl.go

```diff
@@ -266,6 +281,39 @@ func evaluateReadOnlyState() bool {
 	return false
 }
 
+func (cmi *ClusterManagerImpl) punchHeartBeat(rvList []dcache.RawVolume) {
+	hostname, err := os.Hostname()
```

Comment:
we can store hostname also in ClusterManagerImpl, populated once in start().
Then the cluster manager can export functions for others to query the nodeid, hostname, IPAddr, etc